File: analyze.py
import re
from urllib.parse import unquote
import os

with open('c:/Users/user/Documents/antrizy/rendered_index.html', 'r', encoding='utf-8') as f:
    content = f.read()

# Find image urls
urls = re.findall(r'https://static\.wixstatic\.com/media/([a-zA-Z0-9_\-\.\~]+(?:%20[a-zA-Z0-9_\-\.\~]+)*)', content)
print('Total wix media urls found:', len(urls))
print('Unique wix media urls:', len(set(urls)))

for u in list(set(urls))[:10]:
    print(' -', u)

css_links = re.findall(r'<link[^>]*rel=[\"\'\s]*stylesheet[\"\'\s]*[^>]*href=[\"\'\s]*([^\"\'>\s]+)[\"\'\s]*', content)
print('\nCSS links:', len(css_links))
for c in css_links:
    print(' -', c)

scripts = re.findall(r'<script[^>]*src=[\"\'\s]*([^\"\'>\s]+)[\"\'\s]*', content)
print('\nScripts:', len(scripts))
for s in scripts[:5]:
    print(' -', s)

# check how images map
import glob
images = glob.glob('c:/Users/user/Documents/antrizy/images/*')
images_basenames = [os.path.basename(img) for img in images]
print('\nTotal images downloaded:', len(images_basenames))
print('Sample downloaded images:', images_basenames[:10])

File: analyze.js
const fs = require('fs');

const content = fs.readFileSync('c:/Users/user/Documents/antrizy/rendered_index.html', 'utf-8');

// Find all image tags and print their attributes to see if we can find original filenames
const imgTagRegex = /<img[^>]+>/g;
let match;
let count = 0;
while ((match = imgTagRegex.exec(content)) !== null && count < 20) {
    console.log(match[0]);
    count++;
}

// Find wix-image custom elements
const wixImageRegex = /<wix-image[^>]+>/g;
count = 0;
while ((match = wixImageRegex.exec(content)) !== null && count < 10) {
    console.log(match[0]);
    count++;
}

// Search for one of the downloaded images in the HTML text
console.log('\nSearching for downloaded image names in HTML:');
const imagesDir = 'c:/Users/user/Documents/antrizy/images';
const downloadedImages = fs.readdirSync(imagesDir);

for (let i = 0; i < 20; i++) {
    const imgName = downloadedImages[i];
    if (content.includes(imgName)) {
        console.log('Found exact match for:', imgName);
    }
    const baseName = imgName.split('.')[0].replace('_edited', '');
    if (content.includes(baseName) && baseName.length > 3) {
        console.log('Found partial match for:', baseName);
    }
}
